easybuild/easyconfigs/k/Kraken2/Kraken2-2.0.8-beta-foss-2018b-Perl-5.28.0.eb

```diff
@@ -0,0 +1,51 @@
+easyblock = 'PackedBinary'
+
+name = 'Kraken2'
+version = '2.0.8-beta'
+versionsuffix = '-Perl-%(perlver)s'
+
+homepage = 'http://www.ccb.jhu.edu/software/%(namelower)s/'
+description = """Kraken is a system for assigning taxonomic labels to short DNA sequences,
+ usually obtained through metagenomic studies. Previous attempts by other
+ bioinformatics software to accomplish this task have often used sequence
+ alignment or machine learning techniques that were quite slow, leading to
+ the development of less sensitive but much faster abundance estimation
+ programs. Kraken aims to achieve high sensitivity and high speed by
+ utilizing exact alignments of k-mers and a novel classification algorithm."""
+
+# part is compiled with $CXX, the rest is in Perl
+toolchain = {'name': 'foss', 'version': '2018b'}
+toolchainopts = {'openmp': True, 'cstd': 'c++11'}
+
+github_account = 'DerrickWood'
+source_urls = [GITHUB_LOWER_SOURCE]
+sources = ['v%(version)s.tar.gz']
+patches = ['%(name)s-%(version)s_CXX-CXXFLAGS.patch']
```

Comment:
@pescobar Could you make sure the patch is available for this version (or specify the patch for the previous version if it's still valid)?